sphinxcontrib/reviewbuilder/writer.py

```diff
@@ -51,10 +57,14 @@ def __init__(self, builder):
         # type: (ReviewBuilder) -> None
         writers.Writer.__init__(self)
         self.builder = builder
-        self.translator_class = self.builder.translator_class or ReVIEWTranslator
```

Comment:
this will lead unexpected behavior. Sphinx-1.6 set TextTranslator to `self.translator_class`